src/BSON/Decimal128.c

```diff
@@ -151,6 +152,25 @@ PHP_METHOD(Decimal128, __toString)
 }
 /* }}} */
 
+/* {{{ proto array Decimal128::jsonSerialize()
+*/
+PHP_METHOD(Decimal128, jsonSerialize)
+{
+	php_phongo_decimal128_t *intern;
+	char                     outbuf[BSON_DECIMAL128_STRING] = "";
```

Comment:
This was previously done in `php_phongo_decimal128_get_properties()` as insurance that the string was null-terminated even if `bson_decimal128_to_string()` decided not to touch it. Based on my reading of `bson_decimal128_to_string()`, does always assign a null-terminated string to its output buffer, but it wasn't documented as doing so and I wanted to be defensive. The alternative was a `memset()` on the array, but I didn't think that was necessary.

`__toString()` and `serialize()` currently use `char outbuf[BSON_DECIMAL128_STRING]` without any assignment, but I wanted to update those later.